client/components/payment-logo/index.jsx

```diff
@@ -12,7 +12,7 @@ import i18n from 'i18n-calypso';
 
 const ALT_TEXT = {
 	alipay: 'Alipay',
-	amex: 'American Express',
+	amex: 'Amex',
```

Comment:
Why? the logo does say American Express and that's consistent with the branding.

Reply:
The `PaymentMethod` was overriding these anyway, figured I'd preserve what was currently being used.

Amex/Visa was a coinflip, discover was a noop, and Mastercard is definitely lower case c now. MasterCard is their old branding.

Reply:
Mastercard/Visa is indeed better, but let's keep American Express for Amex and fix elsewhere if needed.

Reply:
(Happy to change)